### ISSUE TITLE
RadioMaterial has no attribute trainable (occurs when a material is defined after loading from scene xml)

### ISSUE THREAD

**Issue by jrounds** [NVlabs/sionna#217]
This one is a little hard to duplicate without loading a scene xml, but should be easy enough by just defining a custom material in XML and then trying to add it to the scene.

Suppose we are loading an xml with 
```
    <bsdf type="twosided" id="mat-my_material">
		<bsdf type="diffuse">
			<rgb value="0.701101 0.644479 0.485150" name="reflectance"/>
		</bsdf>
	</bsdf>
    <shape type="obj">
        <ref id="mat-my_material" name="bsdf"/>
        <boolean name="face_normals" value="true"/>
        <string name="filename" value="my_file.obj" />
    </shape>
```

load_scene() will place holder my_material in this code:
https://github.com/NVlabs/sionna/blob/main/sionna/rt/scene.py#L1444-L1446

Now suppose you do
```
from sionna.rt import RadioMaterial
from sionna.rt import load_scene
scene = load_scene("my_scene.xml:) 

my_material = RadioMaterial("my_material",
                                relative_permittivity=2.0,
                                conductivity=5.0)
scene.add(my_material)
```

That will error at this line of code:
https://github.com/NVlabs/sionna/blob/main/sionna/rt/scene.py#L344

RadioMaterial does not have attribute  "trainable" (which it doesn't).

Did it used too?  mypy should have caught this.

**Comment by jrounds** [comment]
A suggestion here would be to define a 
```
class RadioMaterial:
    def update(self, r: RadioMaterial)->None:
          """ Updates this object from attributes in r"""
          pass
```

Then instead of https://github.com/NVlabs/sionna/blob/main/sionna/rt/scene.py#L344
call 
```
s_item.update(item)
```

The reason is code smell.  Right now the calling scene has to know how to update RadioMaterials from RadioMaterials, which is basically easy to forget to do.

If the RadioMaterial has a field update then the person modifying RadioMaterial has the duty to write/unit test future attribute updates and it is easier to have it feed into other parts of your code without changing it everywhere.

**Comment by faycalaa** [comment]
Thanks for reporting this issue. It will be fixed in the next release.

**Comment by faycalaa** [comment]
Fixed by v0.16.
